libraries/GCS_MAVLink/GCS_Common.cpp

```diff
@@ -2761,19 +2761,23 @@ void GCS_MAVLINK::send_home_position() const
 
     // get home position from origin
     Vector3f home_pos_neu_cm;
```

Comment:
```suggestion
    Vector3f home_pos_ned;
```

You're nicely converting it immediately after it is fetched, so the name is almost immediately made incorrect :-)

Reply:
Ha! I didn't even notice. I'll change that.

Reply:
done